core/tests/kernel/coverage_test.py

```diff
@@ -0,0 +1,145 @@
+import torch
+import shark_turbine.kernel as tk
+import shark_turbine.kernel.lang as tkl
+import pytest
+
+
+TKL_TO_TORCH_DTYPE = {
```

Comment:
Can we add this to DataType as a method? a from_torch_type and to_torch_type would be good to have.

Reply:
Done. PTAL.